### PR TITLE
add copy/paste plus various standard menu shortcuts

### PR DESCRIPTION
**Pull request by levthedev**
This PR adds copy/paste, quit, hide, undo/redo, etc. One thing I found is that in development, if I CMD-Q, it will give me an error while it's quitting about the child_process (though it still kills the child_process). However, this error does not appear on packaged versions of the app and doesn't seem to affect anything. Thoughts?

### CODE REVIEW

**Comment by andySigler** [comment]
I got this error when loading a json protocol file. The protocol runs, but does not get saved to the otone_data folder.

screen shot 2016-08-03 at 11 56 23 am

**Comment by andySigler** [comment]
The directory used in `loadFiles.js` is [relative to the script](https://github.com/OpenTrons/otone_frontend/blob/master/src/js/loadFiles.js#L259), but the otone_data filepath is set using `electron.getPath('userData') [on this line](https://github.com/OpenTrons/otone_frontend/blob/master/main.js#L72).

**Comment by andySigler** [comment]
Added `remote.getGlobal()` to `loadFile.js` for sharing the `otone_data` folder location. This PR works well built on my mac, looks good to me 👍

**Comment by levthedev** [comment]
Oops! Weirdly enough I tested it on Windows and Mac builds yesterday and it
worked ok - I just left for Ikea but I'll fix it when I get back.

File: main.js [hunk 1]
```diff
@@ -71,6 +67,8 @@ function execFile(filePath, extraArgs) {
 function startBackend() {
     const userDataPath = app.getPath('userData');
 
+    global.userDataPath = userDataPath;
```
Comment (SimplyAhmazing) [inline main.js]: Just change this to not be global and I'd say everything else seems fine to me.

**Comment by SimplyAhmazing** [comment]
I fixed the Cmd-Q `Uncaught Exception` error in my branch.

**Comment by andySigler** [comment]
I'm going to go ahead and merge this in

**Pull request merged**